ycms/cms/templates/zentral.html

```diff
@@ -0,0 +1,86 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% block content %}
```

Comment:
```suggestion
{% extends "_base.html" %}
{% load i18n %}
{% load static %}
{% load widget_tweaks %}
{% block title %}
    {% translate "Ward Management" %}
{% endblock title %}
{% block content %}
```
THis just makes the browser tab title complete, instead of just reading "Ycms | ".